@here/harp-mapview/test/ThemeLoaderTest.ts

```diff
@@ -306,6 +306,48 @@ describe("ThemeLoader", function () {
             assert.deepEqual(result.definitions!.roadColor, { type: "color", value: "#fff" });
             assert.deepEqual(result.definitions!.waterColor, { type: "color", value: "#0f0" });
         });
+        it("supports multiple inheritance with textures", async function () {
+            const inheritedTheme: Theme = {
+                extends: [
+                    {
+                        images: {
+                            foo: {
+                                url: "icons://maki_icons.png",
+                                preload: true
+                            }
+                        },
+                        imageTextures: [
+                            {
+                                name: "foo",
+                                image: "foo"
+                            }
+                        ]
+                    },
+                    {
+                        images: {
+                            bar: {
+                                url: "icons://maki_icons.png",
+                                preload: true
+                            }
+                        },
+                        imageTextures: [
+                            {
+                                name: "bar",
+                                image: "bar"
+                            }
+                        ]
+                    }
+                ]
+            };
+
+            const result = await ThemeLoader.load(inheritedTheme);
+            assert.exists(result.images?.foo);
+            assert.exists(result.images?.bar);
+            assert.deepEqual(result.imageTextures?.map(imageTexture => imageTexture.name).sort(), [
```

Comment:
Please change one of the imageTextures to have a collision and check that the second one overrides the first one.

Reply:
Good point. Done.

Reply:
Sorry, didn't mean to resolve.

It is better, but if I understand correctly, the test still doesn't test that the collision is resolved correctly because the "baz" points to the same image as the base theme. So you can't say for certain that you have the ImageTexture from the theme.

If you change the `image:"baz"` to something different (but keep the "name" the same), then you can be sure that the collision is correctly resolved.

Reply:
Nevermind, I read the code wrong. Resolving and will approve.